packages/mui-system/src/createTheme/applyStyles.ts

```diff
@@ -11,34 +11,28 @@ export interface ApplyStyles<K extends string> {
  *  - [CSS theme variables](https://mui.com/material-ui/customization/css-theme-variables/overview/)
  *  - Zero-runtime engine
  *
- * Tips: Use an array over object spread and place `theme.applyStyles()` last.
- *
- * ✅ [{ background: '#e5e5e5' }, theme.applyStyles('dark', { background: '#1c1c1c' })]
- *
- * 🚫 { background: '#e5e5e5', ...theme.applyStyles('dark', { background: '#1c1c1c' })}
- *
  * @example
  * 1. using with `styled`:
  * ```jsx
- *   const Component = styled('div')(({ theme }) => [
- *     { background: '#e5e5e5' },
- *     theme.applyStyles('dark', {
+ *   const Component = styled('div')(({ theme }) => {
+ *     background: '#e5e5e5',
+ *     ...theme.applyStyles('dark', {
  *       background: '#1c1c1c',
  *       color: '#fff',
  *     }),
- *   ]);
+ *   });
  * ```
  *
  * @example
  * 2. using with `sx` prop:
  * ```jsx
- *   <Box sx={theme => [
```

Comment:
This doesn't work, sx prop doesn't accept `(theme) => array` as styled does. I removed the tip and used the same format for all of them.